### DISCUSSION TITLE
Contributing with new schematics and diagrams

### DISCUSSION THREAD

**Discussion by Gustavomurta** [probonopd/MiniDexed#518] [Ideas]
I'm building a miniDexed with Raspberry Pi Zero and would like to contribute some diagrams that I've edited. 
If interested, I can redraw it with English subtitles.

https://github.com/Gustavomurta/miniDexed_RaspPi_synthesizer/tree/main/Hardware

For example:
https://github.com/Gustavomurta/miniDexed_RaspPi_synthesizer/blob/main/Hardware/Display_LCD/LCD16x2%20I2C%20minDexed.JPG
https://github.com/Gustavomurta/miniDexed_RaspPi_synthesizer/blob/main/Hardware/Encoder/Encoder%20Rotativo%20miniDexed.JPG
https://github.com/Gustavomurta/miniDexed_RaspPi_synthesizer/blob/main/Hardware/PCM5102/PCM5102%20Raspberry%20Pi%20diagrama.JPG

**Comment by Gustavomurta** [comment]
![LCD16x2 I2C minDexed](https://github.com/probonopd/MiniDexed/assets/4587366/9c794896-87b3-420d-82e7-fabb81d1c48b)

**Reply by Gustavomurta** [reply]
https://github.com/Gustavomurta/miniDexed_RaspPi_synthesizer/tree/main/Hardware/Display_LCD

**Comment by Gustavomurta** [comment]
![Encoder Rotativo miniDexed](https://github.com/probonopd/MiniDexed/assets/4587366/dc47e518-cf55-4ee3-90ab-9f64d853c994)

**Reply by Gustavomurta** [reply]
https://github.com/Gustavomurta/miniDexed_RaspPi_synthesizer/tree/main/Hardware/Encoder

**Comment by Gustavomurta** [comment]
![PCM5102 Raspberry Pi diagrama](https://github.com/probonopd/MiniDexed/assets/4587366/8f3ab5a3-9991-4c37-bf09-10c8a472540c)

**Reply by Gustavomurta** [reply]
https://github.com/Gustavomurta/miniDexed_RaspPi_synthesizer/tree/main/Hardware/PCM5102

**Comment by Gustavomurta** [comment]
![MiniDexed amplifier](https://github.com/probonopd/MiniDexed/assets/4587366/864e0730-c546-46de-84b2-8c70702f1ad0)

**Reply by Gustavomurta** [reply]
MiniDexed amplifier and 5V Regulator
PAM8403 + LM2596

https://github.com/Gustavomurta/miniDexed_RaspPi_synthesizer/tree/main/Hardware/Audio_amplifier

**Comment by Gustavomurta** [comment]
Another diagram made by me. 
miniDexed MIDI IN circuit:


References:
https://www.theatreofnoise.com/2017/08/midi-wiring-diagrams.html
https://www.midi.org/specifications/midi1-specifications/midi-1-0-core-specifications/midi-1-0-detailed-specification-2


![miniDexed MIDIN edited2](https://github.com/probonopd/MiniDexed/assets/4587366/904aa482-cef7-4de1-b868-89a3239f5eb0)

**Reply by Gustavomurta** [reply]
H11L1 pinout:

![H11L1b](https://github.com/probonopd/MiniDexed/assets/4587366/1312ab9c-dad8-4573-9d6b-c7accb36f668)


![MIDI-female-wire](https://github.com/probonopd/MiniDexed/assets/4587366/24734eff-8ac7-460b-a272-9553a34f9ed3)

**Comment by seymourgler** [comment]
Awesome work, thanks for sharing! I notice your diagram is mostly identical to the Classic MIDI Input diagram on the main MiniDexed Hardware Wiki, recommended by @diyelectromusic (but yours is more clear and concise, IMHO.)

However, that other diagram specifies a 100nF capacitor between the +3.3V and GND, which is confusing to me. Is that a separate, necessary component? Did you use one in your build? If so, how and where does it connect?
![Classic MIDI Input](https://github.com/probonopd/MiniDexed/assets/83792812/5554c5b4-41bb-48fc-9cf8-5ea30019c3b6)

**Comment by Gustavomurta** [comment]
I don't see the need to mount the C1 capacitor, since the 3.3V power supply is drawn from the Raspberry Pi board.
